Purchasing.Mvc/Controllers/OrderController.cs

```diff
@@ -2146,6 +2177,35 @@ public JsonNetResult GetKfsStatus(int id)
             
         }
 
+        public async Task<JsonNetResult> GetAeStatus(int id)
+        {
+            // load the order
+            var order = _repositoryFactory.OrderRepository.GetNullableById(id);
+            try
+            {
+                if (string.IsNullOrWhiteSpace(order.ReferenceNumber))
+                {
+                    return new JsonNetResult(null);
+                }
+                var rtValue = new AeResultStatus();
+                rtValue.AeStatus = await _aggieEnterpriseService.LookupOrderStatus(order.ReferenceNumber.Trim());
+                try
+                {
+                    rtValue.Status = Enum.GetName(typeof(RequestStatus), rtValue.AeStatus.RequestStatus);
```

Comment:
how will this ever throw an exception?  I think that should only be possible if the enum type was invalid, but that's static so we know that won't happen